packages/module/patternfly-docs/content/extensions/component-groups/about-component-groups.md

```diff
@@ -0,0 +1,9 @@
+---
+section: extensions
+subsection: Component groups
+id: About component groups
+--- 
+
+The **patternfly/react-component-groups** extension contains a range of React components that are more complex than basic PatternFly components, often combining and adjusting multiple base components to address common use cases. 
+
+This extension replaces **RedHatInsights/frontend-components**. To migrate your product to this new implementation, you can refer to migration guide details in [the README in the React component groups repository](https://github.com/patternfly/react-component-groups/migration.md).  
```

Comment:
We could add the following describing imports.

## Direct Imports

React component groups supports direct imports.  This is useful in improving your bundle size. To directly import a component your import should look like the following for the component:
`
import Ansible from '@patternfly/react-component-groups/dist/dynamic/Ansible';`

Alternatively could use a plugin to change relative imports to direct imports.   This is specifically important when using fedarated modules in webpack.  To learn more about the importance importing has on your bundle size see the following article:

[Webpack module federation think twice before sharing a dependency](https://medium.com/@marvusm.mmi/webpack-module-federation-think-twice-before-sharing-a-dependency-18b3b0e352cb)